src/test/java/marquez/db/mappers/DatasetRowExtendedMapperTest.java

```diff
@@ -0,0 +1,162 @@
+import java.time.Instant;
+import java.util.UUID;
+import marquez.UnitTests;
+import marquez.common.models.DatasetName;
+import marquez.common.models.DatasetUrn;
+import marquez.common.models.DatasourceUrn;
+import marquez.common.models.Description;
+import marquez.db.Columns;
+import marquez.db.models.DatasetRowExtended;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(UnitTests.class)
+public class DatasetRowExtendedMapperTest {
+  private static final UUID ROW_UUID = newRowUuid();
+  private static final Instant CREATED_AT = newTimestamp();
+  private static final Instant UPDATED_AT = CREATED_AT;
+  private static final UUID NAMESPACE_ROW_UUID = newRowUuid();
+  private static final UUID DATASOURCE_ROW_UUID = newRowUuid();
+  private static final DatasetName NAME = newDatasetName();
+  private static final DatasetUrn URN = newDatasetUrn();
+  private static final DatasourceUrn DATASOURCE_URN = newDatasourceUrn();
+  private static final Description DESCRIPTION = newDescription();
+  private static final UUID CURRENT_VERSION_UUID = newRowUuid();
+
+  @Test
+  public void testMap_row() throws SQLException {
+    final Object exists = mock(Object.class);
+    final ResultSet results = mock(ResultSet.class);
+    when(results.getObject(Columns.ROW_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.CREATED_AT)).thenReturn(exists);
+    when(results.getObject(Columns.UPDATED_AT)).thenReturn(exists);
+    when(results.getObject(Columns.NAMESPACE_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.DATASOURCE_UUID)).thenReturn(exists);
+    when(results.getObject(Columns.NAME)).thenReturn(exists);
+    when(results.getObject(Columns.URN)).thenReturn(exists);
+    when(results.getObject(Columns.DATASOURCE_URN)).thenReturn(exists);
+    when(results.getObject(Columns.DESCRIPTION)).thenReturn(exists);
+    when(results.getObject(Columns.CURRENT_VERSION_UUID)).thenReturn(exists);
+
+    when(results.getObject(Columns.ROW_UUID, UUID.class)).thenReturn(ROW_UUID);
+    when(results.getTimestamp(Columns.CREATED_AT)).thenReturn(Timestamp.from(CREATED_AT));
+    when(results.getTimestamp(Columns.UPDATED_AT)).thenReturn(Timestamp.from(UPDATED_AT));
+    when(results.getObject(Columns.NAMESPACE_UUID, UUID.class)).thenReturn(NAMESPACE_ROW_UUID);
+    when(results.getObject(Columns.DATASOURCE_UUID, UUID.class)).thenReturn(DATASOURCE_ROW_UUID);
+    when(results.getString(Columns.NAME)).thenReturn(NAME.getValue());
+    when(results.getString(Columns.URN)).thenReturn(URN.getValue());
+    when(results.getString(Columns.DATASOURCE_URN)).thenReturn(DATASOURCE_URN.getValue());
+    when(results.getString(Columns.DESCRIPTION)).thenReturn(DESCRIPTION.getValue());
+    when(results.getObject(Columns.CURRENT_VERSION_UUID, UUID.class))
+        .thenReturn(CURRENT_VERSION_UUID);
+
+    final StatementContext context = mock(StatementContext.class);
```

Comment:
super-nit: Maybe this should be declared at the top of the function with the other mock declarations?

Reply:
Yeah, that's cleaner! I'll make the change :)